backends/advanced/src/advanced_omi_backend/utils/audio_chunk_utils.py

```
@@ -395,6 +395,154 @@ async def reconstruct_wav_from_conversation(
+    Note:
+        Overlap is added to all segments except the final one, to ensure
+        speaker continuity across segment boundaries. Overlapping regions
+        should be merged during post-processing.
+    """
+    from advanced_omi_backend.models.conversation import Conversation
+
+    # Get conversation metadata
+    conversation = await Conversation.get(conversation_id)
+
+    if not conversation:
+        raise ValueError(f"Conversation {conversation_id} not found")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)